packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -184,13 +172,13 @@ impl Experiment {
                 proto::EngineStatus::Logs(sim_id, logs) => {
                     for log in logs {
                         if !log.is_empty() {
-                            info!(target: "behaviors", "[{experiment_id}][{sim_id}]: {log}");
+                            info!(target: "behaviors", "[{experiment_name}][{sim_id}]: {log}");
```

Comment:
Agreed, although again I think we need to have a discussion around the strategy for tracing and spans. (As an example, spans are really going to decrease human readability of logging, but greatly increase the power of logging when ingested into datadog or similar. Because of this we might want to keep some information like experiment_name within the logs, *and* within spans)

Reply:
So I'd prefer if we leave this to another PR